### PR TITLE
Community leader Onboarding Revisions 2

### PR DESCRIPTION
**Pull request by robbiecarlton**
screen shot 2016-10-02 at 10 39 13 am
screen shot 2016-10-02 at 10 39 02 am
screen shot 2016-10-02 at 10 38 51 am
screen shot 2016-10-02 at 10 38 34 am

### CODE REVIEW

**Comment by robbiecarlton** [comment]
depends on https://github.com/Hylozoic/hylo-node/pull/173

**Comment by robbiecarlton** [comment]
@levity - good to merge?

File: src/components/CommentForm.js [hunk 1]
```diff
@@ -66,14 +64,23 @@ export default class CommentForm extends React.Component {
...
-      .then(({ error }) => {
-        if (error) return
+      .then(action => {
+        if (responseMissingTagDescriptions(action)) return showTagEditor()
+        if (action.error) return
         trackEvent(ADDED_COMMENT, {post: {id: postId}})
       })
     } else {
       dispatch(updateComment(commentId, text, this.state.tagDescriptions))
+      .then(action => {
```
Comment (levity) [inline src/components/CommentForm.js]: `action => responseMissingTagDescriptions(action) && showTagEditor()` is a nice way to one-line this

File: src/containers/App.js [hunk 2]
```diff
@@ -81,18 +81,22 @@ export default class App extends React.Component {
...
 
       <Notifier messages={notifierMessages}
         remove={id => dispatch(removeNotification(id))}/>
       <LiveStatusPoller community={community}/>
       <PageTitleController/>
       {!isEmpty(tagPopover) && <TagPopover {...{tagPopover}}/>}
-      <ModalWrapper show={get('show', showModal)} params={get('params', showModal)}/>
+      {openModals.map((modal, i) =>
+        <ModalWrapper key={i}
+          top={i === openModals.length - 1}
```
Comment (levity) [inline src/containers/App.js:98]: can you explain how the `top` parameter is supposed to work here? what happens to scrolling for the 2nd, nth, etc. modal?

File: src/components/Modal.js [hunk 3]
```diff
@@ -112,9 +141,9 @@ ModalContainer.propTypes = {
 ModalContainer.contextTypes = {isMobile: bool}
 
 export const Modal = (props, { isMobile }) => {
-  const { children, title, subtitle, onCancel, standalone } = props
+  const { children, title, subtitle, onCancel, standalone, noHeader } = props
   return <ModalContainer {...props}>
-    <div className='title'>
+    {!noHeader && <div className='title'>
```
Comment (levity) [inline src/components/Modal.js]: you could check `(title || subtitle)` here instead of adding a prop

**Comment by levity** [comment]
this is a pretty big diff with lots of interaction. can you set it up behind a feature flag so we can test it internally?

**Comment by robbiecarlton** [comment]
Made those changes

**Comment by levity** [comment]
🆒  -- i've also set the FEATURE_FLAG_COMMUNITY_SETUP_CHECKLIST env var to "testing" in heroku, so everyone in Hylo on Hylo can see this and test it once it's deployed. Let's not forget to change that to "on" when it's been satisfactorily tested :)

**Pull request merged**